src/database/migrations/20240815181652-add-permission-for-actions.js

```diff
@@ -0,0 +1,78 @@
+'use strict'
+
+require('module-alias/register')
+const common = require('@constants/common')
+const Permissions = require('@database/models/index').Permission
+
+const getPermissionId = async (module, request_type, api_path) => {
+	try {
+		const permission = await Permissions.findOne({
+			where: { module, request_type, api_path },
+		})
+
+		if (!permission) {
+			throw permission
+		}
+
+		return permission.id
+	} catch (error) {
+		throw error
+	}
+}
+
+module.exports = {
+	up: async (queryInterface, Sequelize) => {
+		try {
+			//create module
+			const modulesData = [{ code: 'actions', status: 'ACTIVE', created_at: new Date(), updated_at: new Date() }]
+
+			// Insert the data into the 'modules' table
+			await queryInterface.bulkInsert('modules', modulesData)
+
+			//create permission
+			const permissionsData = [
+				{
+					code: 'action_permissions',
+					module: 'actions',
+					request_type: ['POST', 'GET'],
+					api_path: '/scp/v1/actions/*',
+					status: 'ACTIVE',
+					created_at: new Date(),
+					updated_at: new Date(),
+				},
+			]
+
+			await queryInterface.bulkInsert('permissions', permissionsData)
+
+			//create role permission mapping
+			const rolePermissionsData = [
+				{
+					role_title: common.ADMIN_ROLE,
+					permission_id: await getPermissionId('actions', ['POST', 'GET'], '/scp/v1/actions/*'),
+					module: 'actions',
+					request_type: ['POST', 'GET'],
+					api_path: '/scp/v1/actions/*',
+					created_at: new Date(),
+					updated_at: new Date(),
+					created_by: 0,
+				},
+				{
+					role_title: common.ORG_ADMIN_ROLE,
+					permission_id: await getPermissionId('actions', ['POST', 'GET'], '/scp/v1/actions/*'),
```

Comment:
Should the ORG_ADMIN have access to actions apis? 
Since it has no org based flows and common for all organization should we give this access to the org_admins?
@priyanka-TL

Reply:
its same like modules org admin also can add their own actions